app/services/configuration_repository_service.rb

```diff
@@ -170,6 +263,33 @@ def remote_file_a_json_array?(file_path)
     # @!group String Helpers
     #####################################################
 
+    # The login of the fastlane.ci bot account
+    #
+    # @return [String]
+    def bot_user_login
+      github_action(bot_user_client) do
+        return bot_user_client.login
```

Comment:
Explicit returns inside blocks are not okay. I think we should have a rubocop rule to disallow this. AFAIK, this return exits the function, and not the block.

Reply:
Oh yeah, I think we need to use `next bot_user_client.login`

Reply:
Why not just `bot_user_client.login`? I think this syntax is a bit confusing. I think the block should also yield the client passed in.

Something like:
```ruby
github_action(bot_user_client) do |client|
  client.login
end
```